metadata-models/src/main/pegasus/com/linkedin/dataprocess/DataProcessInstanceOutput.pdl

```diff
@@ -15,7 +15,9 @@ record DataProcessInstanceOutput {
   @Relationship = {
     "/*": {
       "name": "Produces",
-      "entityTypes": [ "dataset" ]
+      "entityTypes": [ "dataset" ],
+      "isLineage": true,
```

Comment:
Are we sure we want isLineage : true here? This will cause all data process instances to appear in lineage view.